src/components/AssetMatching/components/SectionHeader/SectionHeader.style.ts

```diff
@@ -38,7 +38,7 @@ function getHeaderBg(isChecked: boolean, isBulkSection: boolean, theme: Theme) {
   }
 
   if (isChecked) {
-    return theme.utils.getColor('branded1', 100, 'normal');
+    return theme.utils.getColor('primary', 100, 'normal');
```

Comment:
`branded1` color on the header's section was a design team's decision and a week ago it was merged to Ictinus as a change request on the initial AssetMatching designs. 
Is the design team informed about these changes? 
We can take it offline and discuss the `branded1` and `branded2` replacement/removal.

Reply:
This is for the `next` version of Ictinus. Not the current one.

Reply:
Ok, nice! Is there a confluence page with the upcoming features/changes that we will add to `next`?